test/ng/directive/inputSpec.js

```diff
@@ -2948,7 +2948,8 @@ describe('input', function() {
       expect(inputElm[1].checked).toBe(true);
       expect(inputElm[2].checked).toBe(false);
 
-      browserTrigger(inputElm[2], 'click');
+      if (event === 'change') inputElm[2].checked = true;
```

Comment:
This is expected, I guess. This is exactly what bootstrap 3 does, see the example code here: https://github.com/angular/angular.js/issues/4516#issue-21221793